sdk/Trace/Clock.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Trace;
+
+use OpenTelemetry\Trace as API;
+
+class Clock implements API\Clock
+{
+    public function moment(): array
+    {
+        $realtime_clock = new RealtimeClock();
+        $monotonic_clock = new MonotonicClock();
```

Comment:
These should be made in the constructor, and possibly passed in instead of calling `new` in the constructor. Maybe as a compromise, accept nullable clocks and if null then make new ones?

Reply:
I think adding these in the constructor does make more sense.  I'll change that.